install/build-docker-images.sh

```diff
@@ -4,9 +4,7 @@ echo ""
 # Build any service that provides the image sentry-self-hosted-local first,
 # as it is used as the base image for sentry-cleanup-self-hosted-local.
 $dcb --force-rm web
-for service in "$($dc config --services)"; do
-  $dcb --force-rm $service
-done
+$dcb --force-rm $($dc config --services)
```

Comment:
I made this change because my IDE warned me about https://github.com/koalaman/shellcheck/wiki/SC2066. I figured since `docker-compose build` accepts multiple arguments, I could simplify. For a quick validation of this, try the following two commands:

```sh
for file in "$(ls)"; do printf "File: $file\n"; done
for file in $(ls); do printf "File: $file\n"; done
```

Given the intention, I will now modify this example to actually build each service separately by removing quoting.